valimail.com.valimail-authenticate.json

```diff
@@ -0,0 +1,38 @@
+{
+  "providerId": "valimail.com",
+  "providerName": "Valimail",
+  "serviceId": "valimail-authenticate",
+  "serviceName": "Valimail Authenticate",
+  "version": 1,
+  "logoUrl": "https://valimail-logos.s3.amazonaws.com/valimail-logo-primary.svg",
+  "description": "Enable Valimail Authenticate on a domain",
+  "variableDescription": "%spftxt% is the SPF record",
+  "records": [
+      {
+      "type": "NS",
+      "host": "_dmarc",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "NS",
+      "host": "_domainkey",
```

Comment:
This one would not allow any other service to use DKIM. 
The record has a structure <key>._domainkey exactly to allow multiple keys per domain.
Also, not every DNS provider supports NS record.
Maybe better to work with CNAME on variable-based host?

Reply:
As described above, this is what we're attempting to achieve with this template. For providers that do not support NS records, we do fall back to CNAME, but that's out of the scope of this template.